util/topgen/intermodule.py

```diff
@@ -1045,6 +1047,7 @@ def get_dangling_im_def(objs: OrderedDict) -> str:
 
     undriven_def = [obj for obj in objs if obj['end_idx'] > 0 and
                     (obj['act'] == 'req' and obj['suffix'] == 'rsp' or
-                     obj['act'] == 'rsp' and obj['suffix'] == 'req')]
+                     obj['act'] == 'rsp' and obj['suffix'] == 'req' or
+                     obj['act'] == 'rcv')]
```

Comment:
Curious: Is this a new possibility or was the code missing it?

Reply:
I think you could argue either way. This is from 239f142fd09, which originally was part of https://github.com/lowRISC/opentitan-integrated/pull/280

Reply:
Okay